FAST-LIO/config/velodyne.yaml

```diff
@@ -1,26 +1,43 @@
-common:
-    lid_topic:  "/velodyne_points"
-    imu_topic:  "/imu/data"
-    time_sync_en: false         # ONLY turn on when external time synchronization is really not possible
-    
-preprocess:
-    lidar_type: 2                # 1 for Livox serials LiDAR, 2 for Velodyne LiDAR, 3 for ouster LiDAR, 
-    scan_line: 32
-    blind: 4
+/**:
+    ros__parameters:
+        reconstructKdTree: true
+        updateState: true
+        updateFrequency: 30
+        max_iteration: 3
+        map_file_path: "./test.pcd"
+        filter_size_corner: 0.5
+        filter_size_surf: 0.5
+        filter_size_map: 0.5
+        cube_side_length: 1000.0
+        point_filter_num: 4
+        feature_extract_enable: false
+        debug_mode_enable: false
+        pcd_save_enable: false
+        visualize_map: true
 
-mapping:
-    acc_cov: 0.1
-    gyr_cov: 0.1
-    b_acc_cov: 0.0001
-    b_gyr_cov: 0.0001
-    fov_degree:    180
-    det_range:     100.0
-    extrinsic_T: [ 0, 0, 0.28] # Lidar center w/r/t IMU
-    extrinsic_R: [ 1, 0, 0, 
-                   0, 1, 0, 
-                   0, 0, 1]
+        common:
+            lid_topic:  "/velodyne_points"
+            imu_topic:  "/mavros/imu/data"
+            time_sync_en: true            # ONLY turn on when external time synchronization is really not possible
```

Comment:
This is false for livox. Should they match?

Reply:
I'm not sure exactly why one is true and one is false but this is basically how it was in the original.

Reply:
okeydoke we can leave as is